source/module_base/complexmatrix.h

```diff
@@ -57,6 +57,9 @@ class ComplexMatrix
 	void set_as_identity_matrix(void);
 
 	std::ostream & print( std::ostream & os, const double threshold_abs=0.0, const double threshold_imag=0.0 ) const;		// Peize Lin add 2021.09.08
+
+	ModuleBase::matrix dble(void);
```

Comment:
What does `dble` mean? No documentation for these functions.

Reply:
i'm actually not using it anymore, will remove it